drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -28,7 +28,12 @@ class Hmis::Hud::Enrollment < Hmis::Hud::Base
 
   # A user can see any enrollment associated with a project they can access
   scope :viewable_by, ->(user) do
-    joins(:project).merge(Hmis::Hud::Project.viewable_by(user))
+    e_t = Hmis::Hud::Enrollment.arel_table
+    p_t = Hmis::Hud::Project.arel_table
+    project_ids = Hmis::Hud::Project.viewable_by(user).pluck(:ProjectID)
+    vieawable_wip = p_t[:project_id].in(project_ids)
+    viewable_enrollment = e_t[:project_id].not_eq(nil).and(e_t[:project_id].in(project_ids))
+    left_joins(wip: :project).where(vieawable_wip.or(viewable_enrollment))
```

Comment:
The `not_eq(nil)` was necessary for a client without any WIPs. Without it, ALL enrollments were returned. Figure out exactly why that was happening and add a test for it.

Reply:
a left_outer_join will create a nil-filled row for any row that doesn't have a wip.  If you use `joins` instead, it will only include those where there is a row in the wips table